.gitignore

```diff
@@ -15,3 +15,18 @@ xcuserdata
 .idea/
 .editorconfig
 */bdk.kt
+
+# Swift related
+.DS_Store
```

Comment:
I wasn't going to request a change for this but in the end I had other comments anyway, so might as well mention this little nit: `.DS_Store` on line 20 is a repeat of line 10.